mapproxy/request/wms/__init__.py

```diff
@@ -227,12 +247,7 @@ def validate_srs(self, srs):
             raise RequestError('unsupported srs: ' + self.params['srs'],
                                code='InvalidSRS', request=self)
     def validate_styles(self):
-        if 'styles' in self.params:
-            styles = self.params['styles']
-            if not set(styles.split(',')).issubset(set(['default', '', 'inspire_common:DEFAULT'])):
-                raise RequestError('unsupported styles: ' + self.params['styles'],
-                                   code='StyleNotDefined', request=self)
-
+        return
```

Comment:
Why have you removed this function?

Reply:
Sorry about this, this is an unrelated change. I removed it from the patch.